src/test/java/spec/yandexSpec/CreateAccount.java

```diff
@@ -18,46 +18,49 @@
 
 public class CreateAccount extends BaseSpec {
     private static final Logger logger = LoggerFactory.getLogger(CreateAccount.class);
-    private static final String EXPECTED_TITLE = "Inbox () — Yandex.Mail";
 
     private HomePage homePage;
     private LogInPage logInPage;
     private MailPage mailPage;
 
     @Test
     public void verifyYandexHomePage() {
-        String ExpectedPageTitle = "Yandex.Mail — free, reliable email";
+        homePage = new HomePage(driver);
 
-        Assertions.assertEquals(driver.getTitle(), ExpectedPageTitle);
+        Assertions.assertTrue(driver.getTitle().contains(ProjectVariables.EXPECTED_TITLE));
```

Comment:
Add message on failure as second parameter in assertion

Reply:
DONE